### PR TITLE
[STK-256] Go build and release

### PR DESCRIPTION
**Pull request by faizanahmad055**

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-1``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/1/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-2``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/2/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-4``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 7](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/7/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 8](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/8/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 9](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/9/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-10``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 11](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/11/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 12](https://jenkins.tools.stackator.com/job/Stakater/job/GitWebhookProxy/job/PR-25/12/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-13``

File: Jenkinsfile [hunk 1]
```diff
@@ -1,135 +1,8 @@
 #!/usr/bin/groovy
-@Library('github.com/stakater/fabric8-pipeline-library@master')
+@Library('github.com/stakater/fabric8-pipeline-library@goBuildAndRelease')
```
Comment (hazim1093) [inline Jenkinsfile]: Use tag `v2.4.0` instead of branch or master

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-14``

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/gitwebhookproxy:PR-25-15``

**Pull request merged**